app/uk/gov/hmrc/helptosavefrontend/connectors/EmailVerificationConnector.scala

```diff
@@ -98,20 +98,20 @@ class EmailVerificationConnectorImpl @Inject() (http: WSHttp, metrics: Metrics)(
 
     status match {
       case BAD_REQUEST ⇒
-        logger.warn(s"[EmailVerification] - received status 400 (Bad Request) (round-trip time: ${nanosToPrettyString(time)})", nino)
```

Comment:
this makes it clear that we received 400 for EmailVerification , instead of a generic warn message `Received status 400 (Bad Request) (round-trip time: ${nanosToPrettyString(time)})`

Reply:
we will be able to see the source of the logging via the `logger` field so we'll always see this message in context

Reply:
yeah, thats what i thought 
LGTM